packages/libs/eda/src/lib/core/components/visualizations/implementations/BipartiteNetworkVisualization.tsx

```diff
@@ -135,6 +147,17 @@ function BipartiteNetworkViz(props: VisualizationProps<Options>) {
     ])
   );
 
+  // Adjust link stroke widths so that it spans an appropriate range for this viz.
+  const dataStrokeWidths =
+    data.value?.bipartitenetwork.data.links.map((link) =>
+      Number(link.strokeWidth)
```

Comment:
Curious what the deal is with strokeWidth being a string. I know that's the API/io-ts definition.

I guess it's nothing to do with dates.

Do we need to protect against non-numbers? Are there expected non-number values? Or maybe just a comment. Just worried about it evaluating to `NaN` at runtime at some point.

Reply:
Its a good question. We've been generally just converting numbers to strings for transferring the data lately in order to retain precision. But in this specific case the strokeWidth should always be a number. I left a comment!

I'm remembering you made a comment in a separate PR that instead of strokeWidth it should really be linkWeight. I think i like that and need to check out how wide reaching a change that would require. But hopefully that would clarify this spot as well!